src/components/Dashboard/FilingHistoryList/StaffNotation.vue

```diff
@@ -0,0 +1,177 @@
+                <v-list dense>
+                    <v-list-item-group color="primary">
+                        <v-list-item @click="showRegistrarsNotationDialog();expand = !expand">
+                            <v-list-item-title
+                                class="file-correction-item"
+                            >
+                                <span class="app-action">Add Registrar's Notation</span>
+                            </v-list-item-title>
+                        </v-list-item>
+                        <v-list-item @click="showRegistrarsOrderDialog();expand = !expand">
+                            <v-list-item-title
+                                class="add-detail-comment-item"
+                            >
+                                <span class="app-action">Add Registrar's Order</span>
+                            </v-list-item-title>
+                        </v-list-item>
+                        <v-list-item @click="showCourtOrderDialog();expand = !expand">
+                            <v-list-item-title
+                                class="add-detail-comment-item"
+                            >
+                                <span class="app-action">Add Court Order</span>
+                            </v-list-item-title>
+                        </v-list-item>
+                    </v-list-item-group>
+                </v-list>
+            </v-menu>
+        </div>
+    </div>
+</template>
+<script lang="ts">
+import { Component, Prop, Emit, Vue } from 'vue-property-decorator'
+
+// Dialog
+import { AddStaffNotationDialog } from '@/components/dialogs'
+
+@Component({
+  components: { AddStaffNotationDialog }
+})
+export default class StaffNotation extends Vue {
+  private isAddingRegistrarsNotation = false
+  private isAddingRegistrarsOrder = false
+  private isAddingCourtOrder = false
+  private expand = false
+
+  /** Prop for the scrollbar offset to be added. */
+  @Prop() private addScrollbarOffset: string
+
+  showRegistrarsNotationDialog (): void {
+    this.isAddingRegistrarsNotation = true
+  }
+  hideRegistrarsNotationDialog (needReload: boolean): void {
+    this.isAddingRegistrarsNotation = false
+    this.close(needReload)
+  }
+  showRegistrarsOrderDialog (): void {
+    this.isAddingRegistrarsOrder = true
+  }
+  hideRegistrarsOrderDialog (needReload: boolean): void {
+    this.isAddingRegistrarsOrder = false
+    this.close(needReload)
+  }
+  showCourtOrderDialog (): void {
+    this.isAddingCourtOrder = true
+  }
+  hideCourtOrderDialog (needReload: boolean): void {
+    this.isAddingCourtOrder = false
+    this.close(needReload)
+  }
+
+  @Emit('close')
+  private close (needReload: boolean): void {
+    // Intentionally empty
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "@/assets/styles/theme.scss";
+#add-staff-filing-label {
+    padding-right: 0.725rem;
+    font-size: 0.875rem;
+    color: $app-blue;
+}
+
+#staff-notation {
+    padding-right: 1.5rem;
```

Comment:
If possible, move layout/whitespace styles into the template and use the Vuetify helper classes. In my experience it's much easier to understand and update the layout if such styles are right in the template (instead of trying to find which CSS is affecting it).